lib/chef/knife/bootstrap/chef-full.erb

```diff
@@ -19,32 +27,128 @@ exists() {
   fi
 }
 
+# do_wget URL FILENAME
+do_wget() {
+  echo "trying wget..."
+  wget -O "$2" "$1" 2>/tmp/stderr
```

Comment:
Yeah, I was worried about that. I need to dig into the usual outliers -- AIX and Solaris 10 -- to see if they have mktemp available. If so, I'll probably call it a day and do that, otherwise we'll have to do something awful like `/tmp/stderr.$$`